### ISSUE TITLE
Network Tab in Docker only displays N/A

### ISSUE THREAD

**Issue by HerrHofrat** [pi-hole/FTL#919]
I think I found an issue with FTL in the official docker image. The network table seems to be populated with [ip neigh show](https://github.com/pi-hole/FTL/blob/b522646b8812b3f43f0935369f6a037189209c6c/src/database/network-table.c#L299) - however, in case the docker container is not started with the --net=host option, the `ip neigh show` command only sees the docker gateway

```
root@pihole2:/# ip neigh show
172.21.0.1 dev eth0 lladdr 02:42:db:65:eb:0e REACHABLE
```

This results in a list that only displays N/A for hardware address and interface
![Bildschirmfoto vom 2020-11-11 13-05-36](https://user-images.githubusercontent.com/1681077/98809898-9d344780-241e-11eb-8d01-7912159bfe81.png)

Is there maybe a workaround to still display the values?

Test environment:
Raspberry PI 4/8GB with the latest Raspbian 64 
Latest Pihole docker image with Pihole 5.1.2 and FTL 5.2

**Comment by DL6ER** [comment]
No. When this information is not available to FTL, it will show `N/A` by design.

**Comment by dschaper** [comment]
If you did not use Host networking mode for Docker then you have a bridge and the Docker container is on a different broadcast domain. Much like a DHCP relay is needed in those networking modes, FTL would need something that forwards the MAC addresses to the isolated bridge.

**Comment by DL6ER** [comment]
I'm moving this issue into the docker namespace as it seems a lot more docker- than FTL-related. Note that EDNS(0) could be used to forward IP and MAC information from the original network to FTL. However, that'd require a ECS-capable FND forwarder which is just another moving gear.
